scripts/upload.sh

```diff
@@ -35,13 +35,7 @@ s3_upload()
     echo ""
     verbose_item "Uploading '$1'"
     # aws s3 cp "$DRY_RUN" --region us-west-1 "$1" "$REMOTE_PATH/$1"
-    aws s3 cp "$1" "$REMOTE_PATH/$1" --region us-west-1
-    if [ -z "$DRY_RUN" ]
-    then
-      curl -fs -XDELETE "https://cdn.auth0.com/js/$LIBRARY_NAME/$VERSION/$1"
-    else
-      echo "(dryrun) curl -fs -XDELETE https://cdn.auth0.com/js/$LIBRARY_NAME/$VERSION/$1"
-    fi
+    aws s3 cp "$1" "$REMOTE_PATH/$1" --region us-west-1 --cache-control "public, max-age=84600"
```

Comment:
What about version vs. wildcard version (https://github.com/auth0/auth0-guardian.js/blob/master/scripts/ci.sh#L98) vs https://github.com/auth0/auth0-guardian.js/blob/master/scripts/ci.sh#L95 they will have the same cache time (23.5h) I'm fine with that, but was that what you intended?